pkg/flowkit/services/transactions.go

```diff
@@ -73,13 +72,21 @@ func (t *Transactions) GetStatus(
 		t.logger.StartProgress("Waiting for transaction to be sealed...")
 	}
 
-	result, err := t.gateway.GetTransactionResult(tx, waitSeal)
-
+	result, err := t.gateway.GetTransactionResultByID(id, waitSeal)
```

Comment:
Using GetTransactionResultByID instead so that we search for the transaction object directly by using the id provided through the argument. 

Current implementation using GetTransactionResult means that we are getting the transaction object from the given id in the argument and then using the transaction object to derive the id. In the case of lilico api, they do not result in the same value of ids